sessions-core/src/main/java/com/adyen/checkout/sessions/interactor/SessionInteractor.kt

```diff
@@ -33,7 +34,8 @@ import kotlinx.coroutines.flow.update
 class SessionInteractor(
     private val sessionRepository: SessionRepository,
     sessionModel: SessionModel,
-    private var isFlowTakenOver: Boolean,
+    @get:VisibleForTesting
```

Comment:
Don't forget to push the change we discussed ;)

Reply:
Done!